### PR TITLE
25 create layout

### PR DESCRIPTION
**Pull request by fadeomar**
#### Issues this PR relates to
#25 
=======

_NOTE: REMEMBER NOT TO REVIEW UNLESS ALL ACCEPTANCE CRITERIA HAS BEEN TICKED OFF_

---

#### Any info the reviewer needs to know to test 
e.g. Add env vars, run npm install

---

#### Any info the team needs to consider before merged into development
e.g. Will a database need to be rebuilt (and is this ok if existing data), add env vars

---

#### Quick Pull Request Checklist

Please make sure you ensure the following has been done before marking your PR as awaiting-review. For full checklist and process review, please refer to Process Guide (#8)

- Branch is up to date with develop
- Project runs
- You can download and install locally, and run the project locally
- All existing and new features run without bugs
- Checked that features/design works across all agreed browsers + devices
- The new work has been checked with Axe accessibility checker and doesn't throw significant errors  
- Linter is used
- Code is consistent with agreed team style
- Naming of files is consistent with project 
- Tests all pass
- No console logs

### CODE REVIEW

File: client/src/components/Layout/Info/index.js [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+import PropTypes from 'prop-types';
+import * as S from './style';
+import Image from '../../Image';
+
+const Info = ({ children, image = 'welcome1', ...props }) => {
+  return (
+    <S.Wrapper>
+      <S.Content>{children}</S.Content>
+      <S.AssetWrapper>
+        <Image image={image} customStyle={{ width: '100%' }} />
```
Comment (thejoefriel) [inline client/src/components/Layout/Info/index.js:11]: Needs to render a different image (the small version) if in mobile. The small version has more cropping and some minor changes even though it looks similar, so it's best to have two versions

File: client/src/components/Layout/General/style.js [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+  ${({ theme }) => theme.media.tablet} {
+    padding: ${({ theme: { spacings } }) =>
+      `${spacings[7]} ${spacings[9]} ${spacings[0]} ${spacings[9]}`};
+  }
+  ${({ theme }) => theme.media.mobile} {
+    padding: ${({ theme: { spacings } }) => `${spacings[6]} 37px 0 38px`};
+  }
+`;
+
+export const Border = styled.div`
```
Comment (thejoefriel) [inline client/src/components/Layout/General/style.js:18]: Generally this is fine but it does create an extra element which can make it less accessibility friendly. Could be worth using :before or :after on the header itself

**Review by thejoefriel — CHANGES_REQUESTED** [review]
Hey @fadeomar since you've started putting graphics in, I'd suggest it's worth finishing it, so can you assign yourself to this issue please and complete the checklist

- [x] Only welcome3 (the example image you've put in) seems to be working. All the others can't find the file https://www.loom.com/share/d64ea58c7fb04032a41ed33b42d27644
- [ ] Only 5 images are upload (I believe all the desktop ones). Please add all 7 x Large ones and 8 x Small ones. Label them as such so we can switch (I know the images are very similar but the mobile has more cropping and it'll be easier just to deal with it as a separate image rather than doing overflow hidden stuff etc)

File: client/src/components/Image/index.js [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+  welcome3: Welcome3,
+  welcome4: Welcome4,
+  welcome5: Welcome5,
+};
+
+const StyledImage = styled.img`
+  max-width: 100%;
+  width: ${({ width }) => (width ? `${width}px` : 'auto')};
+  height: ${({ height }) => (height ? `${height}px` : 'auto')};
+  margin: ${({ margin }) => margin || '0'};
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/components/Image/index.js:22]: done

File: client/src/components/Image/index.js [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+`;
+
+const Image = ({ image, width, height, margin, alt, customStyle }) => {
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/components/Image/index.js:25]: done

File: client/src/components/Layout/General/style.js [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+  height: ${({ theme: { spacings } }) => spacings[1]};
+  width: 100%;
+  position: absolute;
+  background: ${({ theme: { gradients } }) => gradients.rainbowHorizontal};
+  ${({ theme }) => theme.media.mobile} {
+    display: none;
```
[the remark this answers is not in the document]
Comment (fadeomar) [reply client/src/components/Layout/General/style.js:24]: done

File: client/src/components/Layout/Info/index.js (hunk 1 above)
Comment (fadeomar) [reply client/src/components/Layout/Info/index.js:11]: done

**Review by thejoefriel — APPROVED** [review]

**Pull request merged**